yt/go/yt/internal/encoder.go

```diff
@@ -959,3 +1037,7 @@ func (e *Encoder) AlterQuery(
 	})
 	return
 }
+
+func EncodeSHA256(input string) string {
```

Comment:
NIT: Looks a bit strange to have this function exported.

Test could have it's own copy I assume.

Reply:
Yes, makes sense since we don't testing this function in the test. Done